e2e/fixtures/citizen/createCase/C100/casePartyDetails/applicantPersonalDetailsContent.ts

```diff
@@ -0,0 +1,33 @@
+export enum ApplicantPersonalDetailsContent {
+  pageTitle = "Provide details for",
+  legend1 = "Have you changed your name?",
+  legend2 = "Gender",
+  legend3 = "Your date of birth",
+  legend4 = "Your place of birth",
+
+  hint1 = "For example, through marriage or adoption or by deed poll. This includes first name, surname and any middle names",
+  hint2 = "For example, 31 3 2016",
+  hint3 = "For example, town or city",
+
+  label1 = "Yes",
+  label2 = "No",
```

Comment:
@GauravChawlaMOJ has suggested common texts like Yes, No, Continue, Day, Month etc all go in `common/commonStaticText.ts`

Reply:
so my issue with that is then you can't use the helper function to iterate through the list, which to me is more efficient